src/ekobox/cards.py

```diff
@@ -0,0 +1,109 @@
+"""Tools to generate runcards."""
+import os
+from math import nan
+from typing import Any, Dict
+
+import numpy as np
+import yaml
+
+from eko import basis_rotation as br
+from eko.io import runcards
+
+Card = Dict[str, Any]
+
+_theory = dict(
+    order=[1, 0],
+    couplings=dict(alphas=[0.118, 91.2], alphaem=[0.007496252, nan]),
+    num_flavs_ref=None,
+    num_flavs_init=None,
+    num_flavs_max_as=6,
+    num_flavs_max_pdf=6,
+    intrinsic_flavors=[4],
+    quark_masses={q: [mq, nan] for mq, q in zip((2.0, 4.5, 173.07), "cbt")},
+    quark_masses_scheme="POLE",
+    matching=[1.0, 1.0, 1.0],
+    fact_to_ren=1.0,
+)
+
+_operator = dict(
+    mu0=1.65,
+    _mugrid=[100.0],
+    configs=dict(
+        evolution_method="iterate-exact",
+        ev_op_max_order=[10, 0],
+        ev_op_iterations=10,
+        interpolation_polynomial_degree=4,
+        interpolation_is_log=True,
+        scvar_method=None,
+        inversion_method=None,
+        n_integration_cores=0,
+    ),
+    debug=dict(
+        skip_singlet=False,
+        skip_non_singlet=False,
+    ),
+    rotations=dict(
+        xgrid=np.geomspace(1e-7, 1.0, 50).tolist(),
+        pids=list(br.flavor_basis_pids),
```

Comment:
Ok, might be weird, but it is not wrong. This is a problem with the runcard: if you do not want `pids` to be in the `OperatorCard`, please open a separate PR based on this one.
I would say that it should be easier to fix, but it might propagate in a few places. Since it is weird but not completely wrong (most of the time you just use the example one, that has the correct basis), then I'd rather not consider blocking for this PR.

Can you open the further PR yourself? @felixhekhorn